control-protocol.md

```diff
@@ -317,10 +339,8 @@ address if it is a multicast module or unicast if UDP unicast analysis.
   # re-generated at program start
   "id" : "hostname=uuid",
 
-  # a sender may send several request in a row. To address the right one
-  # the reply host will send back the sequence number.
-  #
-  # A receiver MUST answer to one equest exactly once.
+	# A sender may send several request in a row.  # A receiver MUST answer to
+  # one equest exactly once.
   #
   # Sequence numbers are message specific. For example: info request message
   # numbers start with 0, later module-start-request first packet also has
```

Comment:
outdated: that has to be removed if we use monotonically increasing sequence numbers